Automattic/jetpack
# CRM: Change customer references to contact

## Proposed changes:

* This PR changes any reference of 'customers' to 'contact', unless within a WooCommerce or ecommerce setting.
* One thing to note, is that it doesn't change pre-existing  statuses of 'Customer' - for example on the Companies Settings page - `/wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=companies`, as these are stored in the DB. Default inputted entries are likely 'Lead,Customer,Refused,Blacklisted' on that page, whereas the text saying what the default entries are will now say 'Lead,Contact,Refused,Blacklisted'. This is the same for regular contacts: `wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=fieldoptions`. This determines the type of contact that is available in the Contact Details 'Status' dropdown when creating a new contact, and on the Client Portal's 'Only generate users for statuses' at `admin.php?page=zerobscrm-plugin-settings&tab=clients`.



### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/2910

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this, using the Jetpack Beta plugin on a test site with this branch applied or testing locally with the branch checked out, you shouldn't see any occurrences of 'customer' (unless in relation to Woo or ecommerce), except for the caveat mentioned in the description above.
* This can be checked on various pages throughout the CRM dashbaord, such as in the welcome wizard , on contacts and companies pages, on the add new quote and add new transaction pages, on the System assistant, in CRM settings, and within customer / contact API status strings.
